tests/driver_pir/main.c

```diff
@@ -30,6 +30,7 @@
 
 static char pir_handler_stack[THREAD_STACKSIZE_MAIN];
 static pir_t dev;
+static const pir_params_t pir_param = {.gpio = PIR_GPIO, .active_high = true};
```

Comment:
don't define params here. Instead, include pir_params.h to use default parameters defined there. This also enables overwriting the active high parameter by CFLAGS commandline parameter. References to the old PIR_GPIO in the driver_pir's Makefile and main.c should then also be adapted to your newly introduced param defines (PIR_PARAM_GPIO and PIR_PARAM_ACTIVE_HIGH)

Reply:
done